accre-internal/Singularity.accre.internal.base

```diff
@@ -7,15 +7,34 @@ Include: yum
   Maintainer Eric Appelt
 
 %help
-  Base container for internal ACCRE use with LMod support
-  To access the software stack make sure to bind mount /cvmfs and /accre
+  Base container for internal ACCRE use with Lmod support
+  To access the software stack make sure to bind mount /accre/arch and /accre/common
```

Comment:
Do you not have to bind to /cvmfs also for the symlinks to work?

Reply:
@appeltel No, because `/cvmfs` is mounted on the host and those two paths are symlinks to it:
```
vanzod@gw341:~$ ll /accre
total 12
drwxr-xr-x 6 root wheel 4096 Jun  6 16:24 admin
lrwxrwxrwx 1 root root    68 May 16 04:19 arch -> /cvmfs/oasis.opensciencegrid.org/accre/mirror/optimized/sandy_bridge
drwxr-xr-x 2 root root  4096 Sep 28  2018 ca
lrwxrwxrwx 1 root root    52 May 16 04:19 common -> /cvmfs/oasis.opensciencegrid.org/accre/mirror/common
lrwxrwxrwx 1 root root    17 Oct  3  2018 usr -> /gpfs22/accre/usr
```
Here is a quick test:
```
vanzod@gw345:~$ singularity shell --bind /accre/arch,/accre/common accre.internal.base.sif 
Singularity: Invoking an interactive shell within container...

Singularity accre.internal.base.sif:/home/vanzod> gcc --version
gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-36)
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Singularity accre.internal.base.sif:/home/vanzod> ml GCC
Singularity accre.internal.base.sif:/home/vanzod> gcc --version
gcc (GCC) 6.4.0
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```